### DISCUSSION TITLE
History

### DISCUSSION THREAD

**Discussion by BiloxiGeek** [jneilliii/OctoPrint-BedLevelVisualizer#602] [General]
Does this plugin save the mesh after it probes for historical purposes or is it overwritten each time? Could be interesting to see a collection of the grids over time, could be put together as a sort of time-lapse video also.

**Comment by jneilliii** [comment]
This plugin doesn't, but the companion plugin I wrote can work with this plugin to save each mesh run and switch between.

https://github.com/jneilliii/OctoPrint-BLVLogger
